app/services/platforms/base-platform.ts

```diff
@@ -60,6 +61,21 @@ export abstract class BasePlatformService<T extends IPlatformState> extends Stat
     );
   }
 
+  protected syncSettingsWithLocalStorage() {
```

Comment:
Can you explain what is happening here?  Why not just use `PersistentStatefulService`?  Is this so we can only store a subset of settings in localStorage?  Would it be worth adding the functionality to only store certain settings when using `PersistentStatefulService` instead?

Reply:
Oh I see now that this is a pattern we were already using in the `FacebookService` and you hoisted it up to the base platform service.  I suppose it's fine for now, we should probably come up with a better pattern at some point though.